### DISCUSSION TITLE
Next.js absolute import issue

### DISCUSSION THREAD

**Discussion by xcodewebsolutions-romil-shah** [vercel/next.js#14768] [Help]
I have implemented next.js absolute import as per documentation here: https://nextjs.org/docs/advanced-features/module-path-aliases.

Its working well in local environment. But when I generate and deploy production build, it is not able to find that virtual path and raises error as provided in screenshot.

Here is the version detail:
ReactJS - 16.9.0
Next.js - 9.4.4
Next/bundle-analyzer - 9.3.6

My jsonconfig.json:
{
  "compilerOptions": {
    "baseUrl": "."
  }
}

![Next js](https://user-images.githubusercontent.com/49144535/86243855-abe33600-bbc4-11ea-809d-07132ea5e613.jpg)

(Module not found: can't resolve 'components/shared/CustomButton'
In 'vercel' build error occured

Error: > build failed because of webpack errors
at build)

Suggest some solution for this issue.

**Comment by timneutkens** [comment]
Please provide a complete reproduction that we can look into.

**Comment by locutermo** [comment]
i have the same error , someone can i help us ?
